felix/fv/qos_controls_test.go

```diff
@@ -156,6 +164,106 @@ var _ = infrastructure.DatastoreDescribe(
 
 			})
 		})
+
+		Context("With packet rate limits", func() {
+			getRules := func(felixId int) string {
+				out, err := tc.Felixes[felixId].ExecOutput("iptables-save", "-c")
+				log.Infof("iptables-save -c output:\n%v", out)
+				Expect(err).NotTo(HaveOccurred())
+				return out
+			}
+
+			It("should limit packet rate correctly", func() {
+				format.MaxLength = 1000000
```

Comment:
I think this is going to affect all following tests.  Personally, I'm a fan of putting `format.MaxLength = 0` somewhere, like in libcalico-go/lib/clientv3/tier_e2e_test.go:
```
func init() {
	// Stop Gomega from chopping off diffs in logs.
	format.MaxLength = 0
}
```

Reply:
(IOW, my only concern is that it _looks_ like this might be a local-only setting, but in fact it won't be.)